samples/testapp/src/commonMain/kotlin/com/android/identity/testapp/ui/ConsentModalBottomSheetListScreen.kt

```diff
@@ -0,0 +1,54 @@
+package com.android.identity.testapp.ui
+
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import com.android.identity.documenttype.knowntypes.DrivingLicense
+
+enum class VerifierType(
+    val description: String
+) {
+    KNOWN_VERIFIER("Known Verifier"),
+    UNKNOWN_VERIFIER_PROXIMITY("Unknown Verifier (Proximity)"),
+    UNKNOWN_VERIFIER_WEBSITE("Unknown Verifier (Website)"),
+}
+
+@Composable
+fun ConsentModalBottomSheetListScreen(
+    onConsentModalBottomSheetClicked: (mdlSampleRequest: String, verifierType: VerifierType) -> Unit,
+    showToast: (message: String) -> Unit
```

Comment:
Looks like we don't use showToast here. Do we need to include it?

Reply:
OK.